src/logger.js

```diff
@@ -17,9 +18,11 @@ export const getLogger: GetLogger = memoize(() => {
 
   return Logger({
     url: loggerUrl,
+    metricNamespacePrefix: BASE_SDK_METRIC_NAMESPACE,
   });
 });
 
+// Deprecated - will remove once we have verified no other clients are importing this function
```

Comment:
option to use jsoc:

```
/** 
 * @deprecated - will remove...
 */
```

and then the message will show on hover in other code bases.

Reply:
oh wow great idea!